CMakeLists.txt

```diff
@@ -435,9 +435,9 @@ if(WITH_CORE)
   # required
   find_package(Proj REQUIRED)
   message(STATUS "Found Proj: ${PROJ_VERSION} ${PROJ_DIR}")
-  if(PROJ_VERSION_MAJOR LESS 8 OR (PROJ_VERSION_MAJOR EQUAL 8 AND (PROJ_VERSION_MINOR LESS 1)))
+  if(PROJ_VERSION_MAJOR VERSION_LESS "8.1")
     message(FATAL_ERROR "Cannot build QGIS using Proj older than 8.1")
-  endif(PROJ_VERSION_MAJOR LESS 8 OR (PROJ_VERSION_MAJOR EQUAL 8 AND (PROJ_VERSION_MINOR LESS 1)))
+  endif(PROJ_VERSION_MAJOR VERSION_LESS "8.1")
```

Comment:
repeating the condition in endif() is ugly and deprecated, especially here since the if() is just 2 lines above